src/main/java/uk/gov/hmcts/darts/task/runner/impl/AbstractLockableAutomatedTask.java

```diff
@@ -80,10 +90,12 @@ public void run() {
                 } else {
                     setAutomatedTaskStatus(AutomatedTaskStatus.SKIPPED);
                     log.warn("Not running task {} now as cron expression has been changed in the database from '{}' to '{}'",
-                             getTaskName(), getLastCronExpression(), dbCronExpression);
+                             getTaskName(), getLastCronExpression(), dbCronExpression
+                    );
                 }
             }
         } catch (Exception exception) {
+            logApi.taskFailed(executionId.get(), getTaskName());
```

Comment:
BTW. This exception is only ever likely to get called when something catastrophic occurs like out of memory etc as most of the automated tasks will handle their own exceptions and in many cases will have their own generic catch of exception and so this will never get reached.